dot/network/service.go

```diff
@@ -232,6 +232,9 @@ func (s *Service) handleMessage(peer peer.ID, msg Message) {
 
 			// handle status message from peer with status submodule
 			s.status.handleMessage(peer, msg.(*StatusMessage))
+
+			s.sendBlockRequestMessage(peer, msg)
```

Comment:
maybe a comment here to explain `sendBlockRequestMessage`

Reply:
sure